plugin-rapl/README.md

```diff
@@ -1,6 +1,52 @@
-# RAPL plugin
+# Plugin RAPL
 
-Collects the energy consumption of the CPU and other RAPL domains.
+The RAPL plugin creates an Alumet **source** that collects measurements of processor energy usage via [RAPL interfaces](https://www.intel.com/content/www/us/en/developer/articles/technical/software-security-guidance/advisory-guidance/running-average-power-limit-energy-reporting.html), such as perf-events and powercap.
 
-Currently, this plugin only works on Linux, because it relies on some abstractions provided by the Linux kernel over RAPL.
-Using MSR registers directly is tricky, hard to maintain, and does not offer any performance benefit.
+## ⚙️ Requirements
+
+- RAPL-compatible processor
+- Linux (the plugin relies on abstractions provided by the kernel - perf-events and powercap)
+- [Set perf_event_paranoid to 0](https://github.com/alumet-dev/packaging/blob/main/docker/README.md#perfmon-usage) (Only for perf-events usage).
```

Comment:
I think we should explain the full story here (e.g. in a subsection): what do the different levels do (0, 1, 2, 4) and what are the capabilities for. Btw, the capabilities are required for the binary too, if you install it manually ([see the user book](https://alumet-dev.github.io/user-book/start/install.html#option-4-installing-from-source)).